specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -6546,6 +6546,11 @@
           "type": "string",
           "readOnly": true
         },
+        "tenantId": {
+          "description": "The id of the tenant in which the product is used.",
```

Comment:
Shall tenant id be populated in all kind of products or just AAD premium. There will be PowerBI and Office in future.
Would it be good to specify that the tenant id will be supported/specified for AAD premium only?

Reply:
This should work for any product with a tenant association.